doc/user_guide/distributed_big_data.rst

```diff
@@ -13,43 +11,46 @@ discuss how to work with large datasets in ``pyxem``.
 .. note::
     If you want more information on working with large datasets in ``pyxem``, please see the
     `Big Data<https://hyperspy.org/hyperspy-doc/current/user_guide/big_data.html>_` section of
-    the `HyperSpy User Guide<https://hyperspy.org/hyperspy-doc/current/user_guide/index.html>`_.
+    the `HyperSpy User Guide<https://hyperspy.org/hyperspy-doc/current/user_guide/index.html>_`.
 
 Loading and Plotting a Dataset
 ------------------------------
 
 Let's start by loading a dataset.  We will use the ``load`` function to load a dataset from
 HyperSpy
 
-.. code-block:: python
+.. code-block::
 
-    >>> import hyperspy.api as hs
-    >>> s = hs.load("big_data.zspy", lazy=True)
-    >>> s
+    import hyperspy.api as hs
+    s = hs.load("big_data.zspy", lazy=True)
+    s
 
 The dataset here is not loaded into memory here, so this should happen instantaneously. We can
 then plot the dataset.
 
-.. code-block:: python
+.. code-block::
 
-    >>> s.plot()
+    s.plot()
 
 Which (in general) will be very slow.  This is because entire dataset is loaded into memory (chuck by chunk)
 to create a navigator image. In many cases a HAADF dataset will be collected as well as a 4-D STEM dataset.
 In this case we can set the navigator to the HAADF dataset instead.
 
 .. code-block:: python
-    >>> haadf = hs.load("haadf.zspy") # load the HAADF dataset
-    >>> s.plot(navigator=haadf) # happens instantaneously
+
+    haadf = hs.load("haadf.zspy") # load the HAADF dataset
+    s.plot(navigator=haadf) # happens instantaneously
 
 This is much faster as the navigator doesn't need to be computed and instead only 1 chunk needs to
 be loaded into memory before plotting!
 
 You can also set the navigator so that by default it is used when plotting.
+
 .. code-block:: python
-    >>> haadf = hs.load("haadf.zspy") # load the HAADF dataset
-    >>> s.navigator = haadf
-    >>> s.plot() # (navigator=haadf) happens instantaneously
+
+    haadf = hs.load("haadf.zspy") # load the HAADF dataset
+    s.navigator = haadf
+    s.plot() # (navigator=haadf) happens instantaneously
```

Comment:
Thank you for including passing a navigator to `plot()` directly. It should be OK to remove the part "(navigator=haadf)" from the comment here as it is described above.